### PR TITLE
Update bundle_ko.properties

### PR DESCRIPTION
**Pull request by AmateurPotion**

### CODE REVIEW

File: core/assets/bundles/bundle_ko.properties [hunk 1]
```diff
@@ -599,8 +618,8 @@ setting.difficulty.insane = 미침
 setting.difficulty.name = 난이도 : 
 setting.screenshake.name = 화면 흔들기
 setting.effects.name = 화면 효과
-setting.destroyedblocks.name = Display Destroyed Blocks
-setting.conveyorpathfinding.name = Conveyor Placement Pathfinding
+setting.destroyedblocks.name = 부서진 블럭 표시
+setting.conveyorpathfinding.name = 컨베이어 설치 보조 기능
 setting.sensitivity.name = 컨트롤러 감도
 setting.saveinterval.name = 저장 간격
 setting.seconds = 초
```
Comment (joshuaptfan) [inline core/assets/bundles/bundle_ko.properties]: This needs a `{0}` somewhere, the English version is

`setting.seconds = {0} Seconds`

with `{0}` being the number of seconds.

**Comment by joshuaptfan** [comment]
On line 157, duplicate variable:
`trace.ip = IP : [accent]{0}{0}`

On line 192:
`server.error = [accent]{0}[crimson]서버를 여는 데 오류가 발생했습니다.`

English version has no variable:
`server.error = [crimson]Error hosting server.`

On line 318:
```
editor.errorload = [accent]{0} 파일을 불러오는 데 실패했습니다.
editor.errorsave = [accent]{0} 파일을 저장하는 데 실패했습니다.
```

English version has no variable:
```
editor.errorload = Error loading file.
editor.errorsave = Error saving file.
```

On line 442:
`zone.unlocked = [LIGHT_GRAY] 잠금 해제되었습니다!`

English version has variable:
`zone.unlocked = [lightgray]{0} unlocked.`

**Comment by Kieaer** [comment]
심각한듯.. 구글 번역미만 스킬사용이후 변수가 모조리 빠진 것 같은데 그냥 저처럼 최신 commit 을 계속 따라갔어야 했을듯

File: core/assets/bundles/bundle_ko.properties (hunk 1 above)
Comment (AmateurPotion) [reply core/assets/bundles/bundle_ko.properties]: oh.

Comment (AmateurPotion) [reply core/assets/bundles/bundle_ko.properties]: thank for your advice

**Pull request merged**